var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -110,6 +111,7 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         ' "default" lets the build_edition make the decision.'
         ' "on" or "off" will always override the build_edition.',
     )
+    variant("ospray", default=False, description="Enable raytracing using osrpray library")
```

Comment:
drop this, the more general raytracing variant is preferred since ParaView can, and eventually will in Spack, support multiple raytracing backends.